src/css/vars.css

```diff
@@ -84,7 +84,7 @@
   --doc-font-size--desktop: calc(17 / var(--rem-base) * 1rem);
   --doc-line-height: 1.6;
   --doc-margin: 0 auto;
-  --doc-margin--desktop: 0 2rem;
+  --doc-margin--desktop: 0 24rem;
```

Comment:
This being set to 24 Rem makes the page look like this:

<img width="1676" alt="image" src="https://github.com/skalenetwork/doc-ui/assets/20197130/f9f8f72a-5ecf-4005-bc71-87a3d1ffd70e">

Suggestion: Revert back to 0 2rem;

Reply:
Alternative - if goal is to center, needed to either define additional breakpoints or change parent sizing possibly?